pkgs/applications/misc/gostatic/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, nix-update-script
+, testers
+, gostatic
+}:
+
+buildGoModule rec {
+  pname = "gostatic";
+  version = "2.34";
+
+  src = fetchFromGitHub {
+    owner = "piranha";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-rdbiIFRZcn9dVaF2anl2iy6FM6boz38vjn+hCpMwcis=";
+  };
+
+  vendorSha256 = "sha256-9YCt9crLuYjd+TUXJyx/EAYIMWM5TD+ZyzLeu0RMxVc=";
+
+  ldflags = [ "-s" "-w" ];
+
+  passthru = {
+    updateScript = nix-update-script { attrPath = pname; };
+    tests.version = testers.testVersion {
+      package = gostatic;
+      command = "gostatic -V";
+      version = version;
```

Comment:
This line can be removed.

Reply:
GitHub's UI is awful -- I meant specifically the `version = version` line, not the test. That should be kept.